src/pybind/mgr/orchestrator/module.py

```diff
@@ -881,13 +881,25 @@ def ukn(s: Optional[str]) -> str:
 
             return HandleCommandResult(stdout=table.get_string())
 
-    @_cli_write_command('orch prometheus access info')
+    @_cli_write_command('orch prometheus set-credentials')
+    def _set_prometheus_access_info(self, user: str, password: str) -> HandleCommandResult:
+        completion = self.set_prometheus_access_info(user, password)
+        result = raise_if_exception(completion)
+        return HandleCommandResult(stdout=json.dumps(result))
+
+    @_cli_write_command('orch alertmanager set-credentials')
+    def _set_alertmanager_access_info(self, user: str, password: str) -> HandleCommandResult:
+        completion = self.set_alertmanager_access_info(user, password)
+        result = raise_if_exception(completion)
+        return HandleCommandResult(stdout=json.dumps(result))
+
+    @_cli_write_command('orch prometheus get-credentials')
     def _get_prometheus_access_info(self) -> HandleCommandResult:
         completion = self.get_prometheus_access_info()
         access_info = raise_if_exception(completion)
         return HandleCommandResult(stdout=json.dumps(access_info))
 
-    @_cli_write_command('orch alertmanager access info')
+    @_cli_write_command('orch alertmanager get-credentials')
```

Comment:
what's the reason for the name change on this command? Afaict, we're just changing how we're storing the info on the backend here, so I don't see why we need a different name to call this. I see elsewhere you are adjusting the dashboard call to this, but that just highlights the exact reason why we generally don't want to change the names. If anyone was using this in some automation we're unaware of, that automation would break.

Reply:
I changed the name to use more explicit `set/get` verbs because `access info` was not clear enough. AFAIK the secure monitoring stack feature is not officially released yet that's why I though the name can be modified without impacting existing users.

Reply:
as discussed in weekly, we decided we're okay with differences between initial reef release and first dot release in this area